apps/epic-web/src/pages/index.tsx

```diff
@@ -63,8 +63,16 @@ const Index: NextPage<{
   const {data: commerceProps, status: commercePropsStatus} =
     trpc.pricing.propsForCommerce.useQuery({
       ...router.query,
-      productId: product.productId,
+      productId:
+        router.query.code || router.query.coupon
+          ? undefined
+          : product.productId,
     })
+
+  const {data: commercePropsProduct} = trpc.products.getProductById.useQuery({
+    productId: commerceProps?.products[0]?.productId,
+  })
```

Comment:
commerce props returns a product id array, so we get the first one

problem: will it return multiple products sometimes?